auto3dseg/algorithm_templates/dints/scripts/train.py

```diff
@@ -54,8 +54,8 @@ def run(config_file: Optional[Union[str, Sequence[str]]] = None, **override):
     fold = parser.get_parsed_content("fold")
     num_images_per_batch = parser.get_parsed_content("training#num_images_per_batch")
     num_epochs = parser.get_parsed_content("training#num_epochs")
-    num_epochs_per_validation = parser.get_parsed_content(
-        "training#num_epochs_per_validation"
+    num_per_validation_epochs = parser.get_parsed_content(
+        "training#num_per_validation_epochs"
```

Comment:
Should it be `num_epochs_per_validation`?

Reply:
current parser cannot understand `num_epochs_per_validation` if we have already defined `num_epochs` in the .json config.